src/cloudai/systems/slurm/strategy/slurm_command_gen_strategy.py

```diff
@@ -145,16 +145,40 @@ def job_name(self, job_name_prefix: str) -> str:
             job_name = f"{self.system.account}-{job_name_prefix}.{datetime.now().strftime('%Y%m%d_%H%M%S')}"
         return job_name
 
+    def _prepare_test_run(self, base_output_dir: Path, tr: TestRun) -> Tuple[Path, "SlurmCommandGenStrategy"]:
```

Comment:
This function make sense to me. Just let's move import to the top of the file, it can be relative.